kuma/static/styles/includes/_vars.scss

```diff
@@ -122,20 +122,16 @@ $diff-font-family: $code-inline-font-family;
 /*
 sizes
 ====================================================================== */
-$h1-font-size: 50px;
-$h2-font-size: 38px;
-$h3-font-size: 24px;
+$h1-font-size: 48px;
+$h2-font-size: 24px;
```

Comment:
I'm not too worried about the big jump, we don't use h1s for anything other than article titles. It's the closeness of the other ones that I think are a problem :( Anyway, discussed it with Kadir and we'll iterate.